package.json

```diff
@@ -4,30 +4,51 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "dev": "docz dev",
-    "clean": "rm -rf .docz/.cache",
-    "build": "docz build",
-    "lint": "eslint src/**/*.js",
-    "serve": "docz serve",
+    "develop": "yarn run test:generate-output && start-storybook -p 6006",
+    "clean": "rm -rf storybook-static",
+    "build": "yarn run test:generate-output && build-storybook --quiet -c .storybook",
+    "lint": "eslint -c .eslintrc.js . && stylelint --config stylelint.config.js 'src/**/*.scss'",
     "test": "jest src",
+    "test:generate-output": "jest --json --outputFile=.jest-test-results.json",
     "test:update": "jest src --updateSnapshot"
   },
   "dependencies": {
-    "docz": "latest",
-    "gatsby": "^2.19.12",
+    "@babel/polyfill": "^7.8.7",
+    "@storybook/addon-a11y": "^5.3.8",
+    "@storybook/addon-docs": "^5.3.8",
+    "@storybook/addon-essentials": "^5.3.8",
+    "@storybook/addon-jest": "^5.3.17",
+    "@storybook/preset-scss": "^1.0.2",
+    "@storybook/react": "next",
+    "classnames": "^2.2.6",
+    "css-loader": "^3.4.2",
+    "gatsby": "^2.18.12",
```

Comment:
Make sure we have the most recent version of Gatsby. The version got downgraded — the caret means it'll likely be the same version, but wanting to check. Wondering if you copied and pasted from another project rather than installing and letting the newest versions write themselves?

Reply:
Yes, this was something that probably occurred when I was trying to figure out the conflicts between the theme and some of our Gatsby projects.